soc/arm/nuvoton_npcx/npcx7/Kconfig.defconfig.series

```diff
@@ -35,6 +35,12 @@ config GPIO_NPCX
 	help
 	  Enable support for NPCX GPIO driver.
 
+config ESPI_NPCX
+	default y
+	depends on ESPI
+	help
+	  Enable support for NPCX ESPI driver.
```

Comment:
This doesn't help much. Ideas to expand this:

- some mention of what eSPI is for
- what features the driver supports
- link to eSPI spec

Reply:
Okay, I know eSPI is not a peripheral interface well known. I will add its spec link of Interl website.

Reply:
Done.